stripecardscan/src/main/java/com/stripe/android/stripecardscan/scanui/SimpleScanStateful.kt

```diff
@@ -0,0 +1,51 @@
+package com.stripe.android.stripecardscan.scanui
+
+import android.util.Log
+import com.stripe.android.camera.framework.AnalyzerLoopErrorListener
+import com.stripe.android.stripecardscan.framework.Config
+
+internal interface SimpleScanStateful {
```

Comment:
let's make this generic to the kind of state. We can define the abstract `ScanState` in this class, but the individual states are specific to the Activities.